recipes-overlayed/kselftests/kselftests_4.12-rc7.bb

```diff
@@ -3,25 +3,27 @@ LICENSE = "GPLv2"
 LIC_FILES_CHKSUM = "file://COPYING;md5=d7810fab7487fb0aad327b76f1be7cd7"
 
 SRC_URI = "\
-    https://www.kernel.org/pub/linux/kernel/v4.x/linux-${PV}.tar.xz \
+    https://git.kernel.org/torvalds/t/linux-${PV}.tar.gz \
```

Comment:
I don't see it published yet:
ERROR: kselftests-4.12-rc7-r0 do_fetch: Fetcher failure for URL: 'https://www.kernel.org/pub/linux/kernel/v4.x/linux-4.12-rc7.tar.xz'. Unable to fetch URL from any source.